template-parts/excerpt/excerpt-chat.php

```diff
@@ -9,7 +9,6 @@
  * @since 1.0.0
  */
 
-// Print the 1st 2 paragraphs, or fallback to the excerpt.
-if ( ! twenty_twenty_one_print_first_instance_of_block( 'core/paragraph', get_the_content(), 2 ) ) {
-	the_excerpt();
-}
+twenty_twenty_one_print_first_instance_of_block( 'core/paragraph', get_the_content(), 2 );
+
+the_excerpt();
```

Comment:
The excerpt here should only be printed if we were unable to find paragraph blocks, so basically for old, non block-based content.